packages/backend/src/Application.ts

```diff
@@ -49,12 +53,18 @@ export class Application {
       safeBlockService,
     })
 
-    const dataSyncService = new DataSyncService()
+    const verifierCollector = new VerifierCollector(
+      ethereumClient,
+      verifierEventRepository
+    )
+
+    const dataSyncService = new DataSyncService(verifierCollector, logger)
     const syncScheduler = new SyncScheduler(
       syncStatusRepository,
       safeBlockService,
       dataSyncService,
-      logger
+      logger,
+      config.core.sync.batchSize
```

Comment:
I'm wondering if we need the nesting. Wouldn't just `config.core.batchSize` be enough?

Reply:
I'm not quite sure if `core.batchSize` is self-documenting enough, but I'll change it to `core.syncBatchSize`.